staging/kube-prometheus-stack/templates/ingress-rbac/grafana.yaml

```diff
@@ -36,11 +36,12 @@ rules:
   - nonResourceURLs:
     - {{ .Values.grafana.ingress.path | trimSuffix "/" }}
     - {{ .Values.grafana.ingress.path | trimSuffix "/" }}/*
+    - {{ .Values.grafana.ingress.path | trimSuffix "/" }}/api/datasources/proxy/*
     verbs:
       - get
       - head
   - nonResourceURLs:
-    - {{ .Values.grafana.ingress.path | trimSuffix "/" }}/api/datasources/proxy/1/api/v1/series
+    - {{ .Values.grafana.ingress.path | trimSuffix "/" }}/api/datasources/proxy/*
```

Comment:
do we still need to allow this route as a POST? I don't see anything mentioning it in the docs 🤔

Reply:
Yes, the POST is a must -- its not on Grafana Doc but I see the error from Chrome DevTool. I guess that's an internal API

Reply:
saw errors /api/datasources/proxy/1/api/v1/series and /api/datasources/proxy/2/api/v1/series (when multiple data sources are available)

Reply:
ah gotcha, thanks!